Changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+**1.4.2**
```

Comment:
@cedvan is this something you can do after merge? These version updates have been really dragging :(